pkg/scheduler/eventhandlers_test.go

```diff
@@ -70,7 +71,9 @@ func TestNodeAllocatableChanged(t *testing.T) {
 			NewAllocatable: v1.ResourceList{v1.ResourceMemory: newQuantity(1024), v1.ResourceStorage: newQuantity(1024)},
 		},
 	} {
+		test := test
 		t.Run(test.Name, func(t *testing.T) {
+			t.Parallel()
```

Comment:
what's the difference if we just keep this line and remove line 51? same question for other occurrences.

Reply:
Apparently this is required to run the Subtests in Parallel. This [post](https://gist.github.com/kunwardeep/80c2e9f3d3256c894898bae82d9f75d0) provides a full explanation of that.

Reply:
what I meant is now that all sub-tests are run in parallel, do we still need parallel the parent test? 

If this is the case, you might consider to add this line to other test for consistency, e.g. `TestFindFitSomeError`

Reply:
Unfortunately, we still need to call the Parallel method from the parent test. Regarding the inconsistency, during my tests I found some test method doesn't support parallelism yet or at least I got runtime errors when I tried to enable it.

Reply:
```
func TestA(t *testing.T) {
  t.Parallel()
}
func TestB(t *testing.T) {
  t.Parallel()
}
```

This means that `TestA` and `TestB` can run in parallel. That's why you see a difference in runtime if you remove the "parent" Parallel.